2-mandatory/5-reading-list.js

```diff
@@ -16,12 +16,37 @@ In this style of testing it is typical to write out as strings exactly what you
 without using any variables or any logic like loops, template strings or if statements.
 */
 
-const books = [];
-  
+const books = [
+  {
+    title: "The Hobbit",
+    author: "J.R.R. Tolkien",
+    alreadyRead: true,
+  },
+  {
+    title: "The Map of Salt and Stars",
+    author: "Jennifer Zeynab Joukhadar",
+    alreadyRead: false,
+  },
+  {
+    title: "Dietland",
+    author: "Sarai Walker",
+    alreadyRead: false,
+  },
+  {
+    title: "A Place for Us",
+    author: "Fatima Farheen Mirza",
+    alreadyRead: true,
+  },
+  {
+    title: "The House of Impossible Beauties",
+    author: "Joseph Cassara",
+    alreadyRead: false,
+  },
```

Comment:
Many people would do  `{title: "The Hobbit", author: "J.R.R. Tolkien", alreadyRead: true,}` all on one line.  But, by separating them on each of their own lines make this code much more readable and is very good practice. Please keep doing this Laleh! Readable code is such a good skill, and you are doing so well!!!